pkg/pulumiyaml/run.go

```diff
@@ -571,7 +582,8 @@ type Evaluator interface {
 
 type programEvaluator struct {
 	*evalContext
-	pulumiCtx *pulumi.Context
+	pulumiCtx     *pulumi.Context
+	runtimeConfig map[string]string
```

Comment:
Both `Runner` and `programEvaluator` have a config option? I think it would be preferred if we could stick with just `Runner` unless they do different things, if so how are they different?